commercetools-models/src/test/java/io/sphere/sdk/products/search/ProductProjectionSearchMainIntegrationTest.java

```diff
@@ -25,13 +31,37 @@
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Ignore
+
 public class ProductProjectionSearchMainIntegrationTest extends ProductProjectionSearchIntegrationTest {
 
     public static final NamedAttributeAccess<String> COLOR_ATTRIBUTE_ACCESS = AttributeAccess.ofString().ofName(ATTR_NAME_COLOR);
     public static final NamedAttributeAccess<Long> SIZE_ATTRIBUTE_ACCESS = AttributeAccess.ofLong().ofName(ATTR_NAME_SIZE);
     public static final ProductProjectionFilterSearchModel PRODUCT_MODEL = ProductProjectionSearchModel.of().filter();
 
+    @Test
+    public void selectAProductByLocaleProjectionInProductProjectionSearch() {
```

Comment:
please add additional tests for the plus and with methods in combination with single strings and lists